.github/workflows/python-checks.yml

```diff
@@ -62,65 +62,6 @@ jobs:
       - name: Dataset unit tests
```

Comment:
nit: not your change but still, we might want to change this to connector unit tests